depends/patches/native_qt/qttools_skip_dependencies.patch

```diff
@@ -0,0 +1,36 @@
+QtTools: Skip unnecessary dependencies:
```

Comment:
I guess same thought here, given that this PR is introducing two copies of this same patch.

Reply:
Fixed in https://github.com/bitcoin/bitcoin/pull/30997#issuecomment-2402990346.